components/Post.js

```diff
@@ -129,9 +82,27 @@ export const Post = (post) => {
 		}
 	}
 
+	async function deletePost() {
+		writeData(
+			`prayer_circle/circles/-NhYtVYMYvc_HpBK-ohk/posts/${post.id}`,
+			null,
+			true
+		);
+		writeData(
+			`prayer_circle/users/BBAzhYq9VGgofMNO5Jl3cmpT2xe2/posts/${post.id}`,
+			null,
+			true
+		);
+		writeData(`prayer_circle/posts/${post.id}`, null, true).then(() => {
+			setTimeout(() => {
+				post.refresh();
+			}, 100);
+		});
+	}
+
 	return (
 		<StyledView className='w-full max-w-[500px]'>
-			<StyledView className='flex flex-col justify-start items-center w-full bg-[#EBEBEB0D] border border-[#6666660D] rounded-[20px] h-auto py-[10px] my-[5px]'>
```

Comment:
Can we add this back? I think the equal padding from the bottom and sides is appealing

Reply:
Thoughts @dev-arekusandoru